app/views/pages/card.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
```

Comment:
@soerenlearnstocode looks good - one thing however: partials always start with an underscore, so this file should be _card.html.erb

Reply:
please update and push again.